lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -48,7 +75,7 @@
     <button class="action action-cancel">
         <%=
             interpolate_text(
-                _.escape(gettext("Cancel {span_start}a new team{span_end}")),
+                _.escape(gettext("Cancel {span_start} team {span_end}")),
```

Comment:
Nice reference there @cptvitamin. :) Throwback for sure, but still relevant. 

In this case we'd like to keep the cancel buttons, in part because it extends Studio's existing patterns, but also because it falls into the sweet spot of usage where cancel buttons actually are #NeilsonApproved. 

While many web applications use pages primarily, from time to time confirmation modals / messages are used to double check actions / operations that might be destructive or accidentally initiated. Neilson describes it this way under the "Cancel: Use sparingly" section. 

"Offer a Cancel button when users may fear that they have committed to something they want to avoid. Having an explicit way to Cancel provides an extra feeling of safety that is not afforded by simply leaving."